javalin/src/test/java/io/javalin/TestStaticFiles.kt

```diff
@@ -157,6 +165,12 @@ class TestStaticFiles {
         assertThat(http.get("/html.html").headers.getFirst(Header.CONTENT_TYPE)).contains("text/html")
     }
 
+    @Test
+    fun `response header always contains content-length if enforceContentLength is true`() = TestUtil.test(configPrecompressionStaticResourceApp) { _, http ->
+        assertThat(!http.get("/secret.html").headers.getFirst(Header.CONTENT_LENGTH).isNullOrEmpty())
```

Comment:
Can you rewrite this to check `isNotBlank` instead of `!isNullOrEmpty` ?

Reply:
Yes!